lib/utils/writeMany.ts

```diff
@@ -0,0 +1,57 @@
+import { JSONObject } from "kuzzle";
+
+export type mRequest = {
+  _id: string;
+  body: JSONObject;
+}
+
+export type mResponse = {
```

Comment:
Those type could be completed and add to the SDK, [this bug](https://github.com/kuzzleio/kuzzle-device-manager/pull/147) would have been avoided for example (not related to this PR)